test/kitchen/site-cookbooks/dd-agent-install/recipes/_install_windows.rb

```diff
@@ -20,6 +20,16 @@
   end
 end
 
+##
+## temporarily disable recv segment coalescing.  This appears to be
+## the root cause of the azure bug causing things to hang.
+## remove this once bug is fixed
+execute 'disable RSC' do
+  command "netsh.exe int tcp set global rsc=disable"
+  #command "disable-netadapterrsc *"
+  ignore_failure true
```

Comment:
Could you add a comment explaining that `ìgnore_failure true` is necessary because this doesn't work on Server 2008R2?